subiquity/controllers/filesystem.py

```diff
@@ -273,15 +285,21 @@ def delete_mount(self, mount):
 
     def create_filesystem(self, volume, spec):
         if spec['fstype'] is None:
-            return
-        fs = self.model.add_filesystem(volume, spec['fstype'])
+            fs = volume.original_fs()
+            if fs is None:
+                return
+            self.model.readd_filesystem(fs)
```

Comment:
read_filesystem? re_add_filesystem? looks like a typo;  maybe update_filesystem()?

Reply:
re-add is definitely the phrase i want to use here (we're putting back a filesystem that has previously been removed. I'll change it to re_add_filesystem though.